app/_how-tos/enable-vault-authentication.md

```diff
@@ -0,0 +1,149 @@
+      icon_url: /assets/icons/gateway.svg
+---
+
+## 1. Create a Consumer
+
+[Consumers](/gateway/entities/consumer/) let you identify the client that's interacting with {{site.base_gateway}}. The credentials will be generated in a later step, so we only need to specify a username.
+
+{% entity_examples %}
+entities:
+  consumers:
+    - username: alex
+{% endentity_examples %}
+
+## 2. Create a Vault object
+
+A Vault object represents the connection between {{site.base_gateway}} and a [Vault](https://www.vaultproject.io/) server. It defines the connection and authentication information used to communicate with the Vault API. This allows different instances of the `vault-auth` plugin to communicate with different Vault servers, providing a flexible deployment and consumption model.
+
+{:.warning}
+> **Important**: The Vault object used for Vault authentication is different from the [Vault entity](/gateway/entities/vault/) used for secrets management.
+
+In this tutorial, we're using `host.docker.internal` as our host instead of the `localhost` that HashiCorp Vault is using because {{site.base_gateway}} is running in a container that has a different `localhost` to you. We'll also use the default `8200` port:
+
+<!--vale off-->
+{% control_plane_request %}
+url: /vault-auth
+method: POST
+headers:
+    - 'Accept: application/json'
+body:
+  name: kong-auth
+  mount: secret
+  protocol: http
+  host: host.docker.internal
+  port: 8200
+  vault_token: root
+  kv: v2
+{% endcontrol_plane_request %}
+<!--vale on-->
+
+Add the value of `id` in the response to your environment, we'll need it in the next step:
+```sh
+export DECK_VAULT_ID=<vault_object_id>
+```
+
+## 3. Enable the Vault Authentication plugin
+
+Enable the [Vault Authentication](/plugins/vault-auth/) plugin, and use the ID of the Vault object to link it to the plugin:
+{% entity_examples %}
+entities:
+  plugins:
+    - name: vault-auth
+      config:
+        vault: 
+          id: ${id}
+
+variables:
+  id:
+    value: $VAULT_ID
+{% endentity_examples %}
+
+## 4. Generate consumer credentials
+
+Use the `POST /vault-auth/{vault}/credentials/{consumer}` endpoint to generate credentials for the Consumer we created:
+<!--vale off-->
+{% control_plane_request %}
+url: /vault-auth/kong-auth/credentials/alex
+method: POST
+headers:
+    - 'Accept: application/json'
+{% endcontrol_plane_request %}
+<!--vale on-->
+
+This request returns an `access_token` and `secret_token`; add these to your environment:
+```sh
+export ACCESS_TOKEN=<consumer_access_token>
+export SECRET_TOKEN=<consumer_secret_token>
+```
+
+## 5. Validate
+
+To validate that the authentication is working as expected, send a request to the Route we created in the [prerequisites](#pre-configured-entities) using the credentials we generated:
+<!--vale off-->
+{% validation request-check %}
+url: '/anything'
+status_code: 200
+headers:
+  - 'access_token: $ACCESS_TOKEN'
+  - 'secret_token: $SECRET_TOKEN'
```

Comment:
```suggestion
  - 'secret_token: $SECRET_TOKEN'
message: OK
```
I think this will get it to render the sentence after this step that confirms what the user should see.